File: Programmers/Lv.1/이상한 문자 만들기/12930.js
function solution(s) {
    let answer = '';
    
    const str = s.split('');
    let upperFlag = true;
    
    for (let i=0; i<str.length; i++) {
        if (upperFlag && str[i] !== ' ') {
            str[i] = str[i].toUpperCase();
            upperFlag = false;
        } else {
            str[i] = str[i].toLowerCase();
            upperFlag = true;
        }
    }
    
    answer = str.join('');
    
    return answer;
}

File: Programmers/Lv.1/이상한 문자 만들기/12930.java
import java.util.Arrays;

class Solution {
    public String solution(String s) {
        String answer = "";
        
        String[] str = s.split("");
        boolean upperFlag = true;
        
        for (int i=0; i<str.length; i++) {
            if (upperFlag && !" ".equals(str[i])) {
                str[i] = str[i].toUpperCase();
                upperFlag = false;
            } else {
                str[i] = str[i].toLowerCase();
                upperFlag = true;
            }
        }
        
        answer = String.join("", str);
        
        return answer;
    }
}
